frontend/fingertips-frontend/playwright/testHelpers.ts

```diff
@@ -212,11 +222,14 @@ export function getScenarioConfig(
       ChartPage.populationPyramidComponent,
       ChartPage.populationPyramidTableComponent,
     ],
+    [`${IndicatorMode.THREE_PLUS_INDICATORS}-${AreaMode.ONE_AREA}`]: [
+      ChartPage.populationPyramidComponent,
```

Comment:
Does this need `populationPyramidTableComponent` to be consistent with the other scenarios?
```suggestion
      ChartPage.populationPyramidComponent,
      ChartPage.populationPyramidTableComponent,
```